packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -39,36 +38,12 @@ export const ConnectedRichTextEditor = (props: ConnectedProps) => {
     () => getPlugins(props.sdk, props.onAction ?? noop, props.restrictedMarks),
     [props.sdk, props.onAction, props.restrictedMarks]
   );
-  const [isFirstRender, setIsFirstRender] = useState(true);
-  const [pendingExternalUpdate, setPendingExternalUpdate] = useState(false);
 
-  const onValueChanged = useOnValueChanged({
-    editorId: id,
-    handler: props.onChange,
-    skip: pendingExternalUpdate || isFirstRender,
-    onSkip: () => setPendingExternalUpdate(false),
-  });
+  const value = toSlateValue(props.value);
 
-  useEffect(() => {
```

Comment:
We now handle this useEffect in the `SyncEditorValue` function?

Reply:
if so does this happen now more than just on first render?

Reply:
Possibly, however, I'm not sure if the old logic _really_ triggered only on the first render. One of these two useEffects must have triggered to propagate `props.value` changes.